overlord/snapstate/backend_test.go

```diff
@@ -94,3 +94,91 @@ func (f *fakeSnappyBackend) Activate(name string, active bool, p progress.Meter)
 	})
 	return nil
 }
+
+func (f *fakeSnappyBackend) CheckSnap(snapFilePath string, flags snappy.InstallFlags) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "check-snap",
+		name: snapFilePath,
+	})
+	return nil
+}
+
+func (f *fakeSnappyBackend) SetupSnap(snapFilePath string, flags snappy.InstallFlags) (string, error) {
+	f.ops = append(f.ops, fakeOp{
+		op:   "setup-snap",
+		name: snapFilePath,
+	})
+	return "some-inst-path", nil
+}
+
+func (f *fakeSnappyBackend) CopySnapData(instSnapPath string, flags snappy.InstallFlags) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "copy-data",
+		name: instSnapPath,
+	})
+	return nil
+}
+
+func (f *fakeSnappyBackend) GenerateSecurityProfile(instSnapPath string) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "generate-security-profile",
```

Comment:
Can we please rename this ``configure-security`` to match naming in other places of interfaces code?